### DISCUSSION TITLE
BMI implementation for JavaScript

### DISCUSSION THREAD

**Discussion by mcflugen** [csdms/bmi#104] [General]
We learned today about a BMI implementation for JavaScript :tada:! I believe it's publicly available but I'm not sure where. @goodalljl may know the answer to this, though.

**Comment by goodalljl** [comment]
Ibrahm Demir and his team at U of Iowa is developing the BMI-JS.  He's on Github @demiribr but I didn't see the project there...

**Comment by demiribr** [comment]
We are still working on some tests, and will release the repo in the coming weeks on @uihilab - https://github.com/uihilab

**Comment by mdpiper** [comment]
Here's the repo: https://github.com/uihilab/BMI-JS
